README.md

```diff
@@ -266,30 +266,38 @@ Object.keys(#{ a: 1, b: 2 })  // ["a", "b"]
 Object.keys(#{ b: 2, a: 1 })  // ["a", "b"]
 ```
 
-If their structure and contents are deeply identical, then `Record` and `Tuple` values considered equal according to all of the equality operations: `Object.is`, `==`, `===`, and the internal SameValueZero algorithm used for Maps and Sets. See further discussion in [#65](https://github.com/rricard/proposal-const-value-types/issues/65).
+If their structure and contents are deeply identical, then `Record` and `Tuple` values considered equal according to all of the equality operations: `Object.is`, `==`, `===`, and the internal SameValueZero algorithm (used for comparing keys of Maps and Sets). They differ in terms of how `-0` is treated:
+- `Object.is` treats `-0` and `0` as unequal
+- `==`, `===` and SameValueZero treat `-0` with `0` as equal
```

Comment:
```suggestion
- `Object.is` (`SameValue`) treats `-0` and `0` as unequal
- `==`, `===`, and `SameValueZero` treat `-0` with `0` as equal

- `Object.is` (`SameValue`) treat `NaN` and `NaN` as equal
- `==`, `===`, and `SameValueZero` treat NaN as unequal
```

Reply:
So, it sounds like this section is unclear. It's trying to talk about how the operators work when dealing with data nested within Records and Tuples: The purpose of this section is to explain how we're making `#[NaN] === #[NaN]`.